src/Auth0.ManagementApi/Models/TenantGuardianMfa.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Newtonsoft.Json;
+
+namespace Auth0.ManagementApi.Models
+{
+    /// <summary>
+    /// Guardian page customization.
+    /// </summary>
+    public class TenantGuardianMfa
```

Comment:
Suggest `TenantGuardianMfaPage` for this (like `TenantErrorPage` above)

Reply:
Will do.